src/library/files/store.ts

```diff
@@ -0,0 +1,66 @@
+import { defineStore } from 'pinia'
+import { FileDirectory } from '@/shared/api'
+
+export const useFilesStore = defineStore('files', {
+  state: () => ({
+    pathID: localStorage.getItem('files_path') || '',
```

Comment:
I think this state should be stored as URL param instead of localstorage/pinia store, otherwise it breaks browser history. There is no need to use as store at all when it's only used by as single component.

Reply:
I do not use browser history for navigation, we have all navigation elements on the page. But if you want, I can change it. But then I will add this option to client settings, because it very useful for me.